frontend/src/structures/stop.ts

```diff
@@ -19,5 +29,26 @@ export default class Stop {
         this.longitude = lng;
         this.created = created;
         this.updated = updated;
+        this.routesOn = [];
+        this.marker = L.marker([this.latitude, this.longitude], {
+            icon: L.icon({
+                iconUrl: StopSVG,
+                iconSize: [12, 12], // size of the icon
+                iconAnchor: [6, 6], // point of the icon which will correspond to marker's location
+                shadowAnchor: [6, 6], // the same for the shadow
+                popupAnchor: [0, 0], // point from which the popup should open relative to the iconAnchor
+              }),
+        });
+    }
+
+    public getMessage(): string {
```

Comment:
Only thing here I would say is to just not display anything when the stop isn't on a route, but also this could be fixed by making sure all stops are up to date in the database, so this is up to your discretion.